### DISCUSSION TITLE
Studies on employability's impact on individuals that contribute to Open Source

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#115] [OSPO Discussions]
Discussion started via [TODO Slack by Thomas Qvidahl](https://thetodogroup.slack.com/archives/C0GBYUWCB/p1653381826667549)

*Does anyone know of any studies done (surveys or research) on the effect on.. hmm let’s call it “employability” for lack of a better word…. of individuals that contribute to Open Source Software? Meaning - if you’re an active contributor to OSS, does it increase your market value as a developer? (I know it DOES, but can we prove it?)*

**Comment by jlprat** [comment]
I only have anecdotal information. But this is how I got my current job.
Also having experience in OSS would open the door to many OSPO positions. For example Aiven's OSPO is looking for several OSS maintainers.

**Comment by LawrenceHecht** [comment]
I can help, just busy right now. I don't think it's worth anybody's time for me to just give a laundry list of surveys.
